homeassistant/components/switch/enigma.py

```diff
@@ -0,0 +1,136 @@
+"""
+homeassistant.components.switch.enigma
```

Comment:
I've just added support to the frontend to turn your media player on/off.

![screen shot 2015-05-30 at 13 50 19](https://cloud.githubusercontent.com/assets/1444314/7899378/3e66bb46-06d5-11e5-9ee5-4be0df9d9bb2.png)